homeassistant/components/automation/__init__.py

```diff
@@ -31,17 +33,77 @@
 
 DEFAULT_CONDITION_TYPE = CONDITION_TYPE_AND
 
+METHOD_TRIGGER = 'trigger'
+METHOD_IF_ACTION = 'if_action'
+
 _LOGGER = logging.getLogger(__name__)
 
 
+def _ensure_list(value):
+    """Wrap value in list if it is not one."""
+    return value if isinstance(value, list) else [value]
```

Comment:
use `list(value)` in case the value is already an iterable type, i.e. tuple. It is a noop when value already is a list, but messes up strings so, `return list(value) if not instance(value, str) else [value]`.